### PR TITLE
`firewall_policy_resource` - support for the `private_ranges` and `allow_sql_redirect`  properties

### PR DESCRIPTION
**Pull request by wuxu92**
and update test code to fix acc test failure.

acc test summary:
--- PASS: TestAccFirewallPolicy_basic (130.12s)
--- PASS: TestAccFirewallPolicy_basicPremium (74.34s)
--- PASS: TestAccFirewallPolicy_complete (71.72s)
--- PASS: TestAccFirewallPolicy_completePremium (439.54s)
--- PASS: TestAccFirewallPolicy_update (192.08s)
--- PASS: TestAccFirewallPolicy_updatePremium (921.11s)
--- PASS: TestAccFirewallPolicy_requiresImport (71.00s)
--- PASS: TestAccFirewallPolicy_inherit (152.48s)
--- PASS: TestAccFirewallPolicy_insights (251.92s)

### CODE REVIEW

File: website/docs/r/firewall_policy.html.markdown [hunk 1]
```diff
@@ -59,6 +59,8 @@ The following arguments are supported:
 
 * `tls_certificate` - (Optional) A `tls_certificate` block as defined below.
 
+* `allow_sql_redirect` - (Optional) Whether SQL Redirect traffic filtering is enabled. Turning on the flag requires no rule using port 11000-11999.
```
Comment (katbyte) [inline website/docs/r/firewall_policy.html.markdown:62]:
```suggestion
* `sql_redirect_allowed` - (Optional) Whether SQL Redirect traffic filtering is allowed. Turning on the flag requires no rule using port 11000-11999.
```

or does it enable it and should be
```suggestion
* `sql_redirect_enabled` - (Optional) Whether SQL Redirect traffic filtering is enabled. Turning on the flag requires no rule using port 11000-11999.
```

**Review by katbyte — CHANGES_REQUESTED** [review]

**Comment by wuxu92** [comment]
hi @katbyte have refactored the schema name

File: website/docs/r/firewall_policy.html.markdown [hunk 2]
```diff
@@ -59,6 +59,8 @@ The following arguments are supported:
 
 * `tls_certificate` - (Optional) A `tls_certificate` block as defined below.
 
+* `sql_redirect_allowed` - (Optional) Whether SQL Redirect traffic filtering is allowed. Turning on the flag requires no rule using port 11000-11999.
```
Comment (katbyte) [inline website/docs/r/firewall_policy.html.markdown:62]:
```suggestion
* `sql_redirect_allowed` - (Optional) Whether SQL Redirect traffic filtering is allowed. Enabling this flag requires no rule using ports between `11000`-`11999`.
```

**Review by katbyte — CHANGES_REQUESTED** [review]
test failures
image

**Comment by wuxu92** [comment]
@katbyte sorry I forgot to change the acc test code, now all tests passed.

image

**Review by katbyte — APPROVED** [review]
LGTM 🧾

**Pull request merged**